packages/purpose-process/src/services/readModelService.ts

```diff
@@ -139,112 +269,11 @@ export function readModelServiceBuilder(
       } satisfies ReadModelFilter<Purpose>);
     },
     async getPurposes(
-      filters: GetPurposesFilters,
-      { offset, limit }: { offset: number; limit: number }
+      filters: ApiGetPurposesFilters,
+      offset: number,
+      limit: number
```

Comment:
Could you make this an object (as it was before)? So there is no confusion about which number is `offset` and which one is `limit`?